test/integration/routes_test.rb

```diff
@@ -11,4 +11,34 @@ class RoutesTest < LegacyIntegrationTest
   should "route to new downtimes controller new downtime" do
     assert_routing("/editions/1/downtime/new", controller: "downtimes", action: "new", edition_id: "1")
   end
+
+  context "allowed content types for new design system" do
+    %i[answer_edition help_page_edition].each do |content_type|
+      context content_type do
+        setup do
+          test_strategy = Flipflop::FeatureSet.current.test!
+          test_strategy.switch!(:design_system_edit, true)
+          @edition = FactoryBot.create!(content_type)
+        end
+
+        should "route to editions controller" do
+          assert_routing("/editions/#{@edition.id}", controller: "editions", action: "show", id: @edition.id.to_s)
+        end
+      end
+    end
+  end
+
+  context "not allowed content types for new design system" do
+    %i[guide_edition local_transaction_edition completed_transaction_edition place_edition simple_smart_answer_edition transaction_edition].each do |content_type|
+      context content_type do
+        setup do
+          @edition = FactoryBot.create!(content_type)
```

Comment:
Should we not be setting the feature toggle on in here as well? Otherwise, we're not really testing the content type check are we?